app/src/components/Map/Timebar.jsx

```diff
@@ -467,6 +471,13 @@ class Timebar extends Component {
   }
 
   render() {
+    const dateFormat = 'DD MMM YYYY';
+    const startDateText = isMobile.phone || isMobile.tablet ? ' start' : 'start date';
```

Comment:
The library relies only on user agents? what about fallback in case of weird user agents? We don't want to have to change the code every time a new startup in china/japan/taiwan/etc releases a new phone. @andresgnlez please spoof user agents and see what happens. While I do get your point that user agents can be more informative, they are also more volatile.